src/libmongoc/src/mongoc/mongoc-change-stream.c

```diff
@@ -247,6 +247,7 @@ _make_cursor (mongoc_change_stream_t *stream)
 
    if (stream->opts.comment.value_type != BSON_TYPE_EOD) {
       bson_append_value (&command_opts, "comment", 7, &stream->opts.comment);
+      bson_append_value (&getmore_opts, "comment", 7, &stream->opts.comment);
```

Comment:
Note to other reviewers: this pertains to https://github.com/mongodb/mongo-c-driver/pull/1014.